packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -376,12 +391,15 @@ class CloudFormation {
 
   describeStack(cfnNestedStackParams, maxTry = 10, timeout = CFN_POLL_TIME) {
     const cfnModel = this.cfn;
+    const log = logger('describeStack.cfn.describeStacks', [cfnNestedStackParams]);
```

Comment:
Will cfnNestedStackParams be always defined? Do we need a check for it?

Reply:
I checked for that and `undefined` does not break the logic